python/bindings.cpp

```diff
@@ -336,7 +336,31 @@ PYBIND11_MODULE(libsonata, m) {
 
         .def("__ne__", &bbp::sonata::operator!=, "Compare selection contents are not equal");
 
-    bindPopulationClass<NodePopulation>(m, "NodePopulation", "Collection of nodes with attributes");
+    bindPopulationClass<NodePopulation>(m, "NodePopulation", "Collection of nodes with attributes")
+        .def(
+            "match_values",
+            [](NodePopulation& obj, const std::string& name, const std::string& value) {
+                return obj.matchAttributeValues(name, value);
+            },
+            "name"_a,
+            "value"_a,
+            "Return selection where the attribute name has values matching value")
+        .def(
+            "match_values",
+            [](NodePopulation& obj, const std::string& name, const int value) {
+                return obj.matchAttributeValues<int>(name, value);
```

Comment:
Anyone have a good idea on how to avoid the repetition of `match_values` w/ `int` and `double`?

Reply:
```python
template <typename ValueType, typename NodePopulationClass> void def_match_values(NodePopulationClass &node){
    node.def(
        "match_values",
        [](NodePopulation& obj, const std::string& name, const ValueType& value) {
            return obj.matchAttributeValues(name, value);
        },
        "name"_a,
        "value"_a,
        "Return selection where the attribute name has values matching value");
}
```

and then

```python

    auto nodePopulation = bindPopulationClass<NodePopulation>(m, "NodePopulation", "Collection of nodes with attributes");
    def_match_values<double>(nodePopulation);
    def_match_values<int>(nodePopulation);
    def_match_values<std::string>(nodePopulation);
```

Reply:
Did basically that.  Still doesn't feel elegant, though.

Reply:
Agreed, but I have nothing better.